api/src/Tests/Cases/AnnouncementTest.php

```diff
@@ -14,25 +14,31 @@ class AnnouncementTest extends CiabTestCase
 
     protected function addAnnouncement($scope): string
```

Comment:
Would it be possible to break this up into separate tests? Test data setup aside, tests should typically be aiming to test and assert against a single function. Descriptive test names would be awesome, too. :)

Reply:
Yeah, this is one of my old test modules where I was just lumping everything together.  Do you think restructure of that should come with this change or separate?

Reply:
It can be separate, although definitely worth pursuing.